x-pack/plugins/lens/public/app_plugin/mounter.tsx

```diff
@@ -15,6 +15,8 @@ import { render, unmountComponentAtNode } from 'react-dom';
 import { i18n } from '@kbn/i18n';
 
 import { DashboardFeatureFlagConfig } from 'src/plugins/dashboard/public';
+import { Provider } from 'react-redux';
+import _ from 'lodash';
```

Comment:
I've seen this in a bunch of places already.
Maybe it won't change much, but would it be better for the bundle size to just import the specific functions in use rather than the whole library?

In this specific case `uniq` and `isEqual` are the only two functions used.